news/4.30/platform.html

```diff
@@ -134,6 +134,22 @@ <h2>General Updates </h2>
       </p>
     </td>
   </tr>
+  <tr id="new-ecf-client">
+    <!-- https://github.com/eclipse-equinox/p2/pull/206 -->
+    <td class="title"><a href="#new-ecf-client">New Http Client Backend for ECF</a></td>
```

Comment:
Isn't this change for P2?
```suggestion
    <td class="title"><a href="#new-ecf-client">New Http Client Backend for P2</a></td>
```

Reply:
P2 Uses ECF what is providing the new backend, we just include it.

Reply:
That's right.
But the relevant part is, that the default used by P2 has changed.
Therefore I would mention that in the headline.